src/main/java/com/thealgorithms/datastructures/lists/SinglyLinkedList.java

```diff
@@ -16,6 +16,19 @@ public class SinglyLinkedList extends Node {
      * Size of SinglyLinkedList
      */
     private int size;
+    
+    //below is the function to find out middle node in the singlyLinkedList.
+      public Node middle(){
+    
+  Node s = head;
+    Node f = s.next;
+    while(f != null && f.next != null){
```

Comment:
If you could make these lines aligned with the existing format, that would be awesome

Reply:
can you tell me, how can i add Juits to this?